testproject/testapp/tests/test_resend_activation.py

```diff
@@ -8,6 +8,7 @@
 from testapp.tests.common import create_user, mock
 
 from djoser.compat import get_user_email
+from djoser.conf import settings as default_settings
```

Comment:
CI fails because of this unused import. Please just remove it and then we're good to go.

Reply:
Removed. Thanks @tomwojcik